src/App.js

```diff
@@ -1,13 +1,109 @@
 import React from 'react';
-
+import uuid from 'uuid';
+import './App.css'
+import Todo from './components/TodoComponents/Todo'
+import TodoForm from './components/TodoComponents/TodoForm'
+import SearchTodo from './components/TodoComponents/searchTodo'
+const todoData = [
+  {
+    task: 'Organize Garage',
+    id: 1528817077286,
+    completed: false
+  },
+  {
+    task: 'Bake Cookies',
+    id: 1528817084358,
+    completed: false
+  }
+];
+const getTodoData= window.localStorage.getItem('todoData');
+console.log(getTodoData)
```

Comment:
Good job on storing data to local storage!
Why line 19 to 28 are here though? You could want to have this logic inside your component,  and use life cycle methods to call your localStorage methods.

Also remember to clean your code from useless console logs.